File: submissions/sm_029_sakshi/week_4/day_1/session_1/checkDisjoint.js


function checkDisjoint(arr1, arr2) {
    var obj = {}
    var isDisjoint = true
    for(var i = 0; i < arr1.length; i++) {
        obj[arr1[i]]=1
    }
    for ( i = 0 ; i < arr2.length ; i++)
    {
        if(obj[arr2[i]] == obj[arr1[i]])
        {
            isDisjoint = false
            
        }
    }

    if(isDisjoint) {
        console.log("true")
    }
    else console.log("false")
}

checkDisjoint ([1,2,3,4,5,6],[7,8,9])

File: submissions/sm_028_sagar/week_13/day_4/session_1/check_disjoint.py
arr1 = ['s','k','a','b']
arr2 = ['d','x','s']

isdisJoint = 'true'

for i in arr1:
    for j in arr2:
        if(i == j):
            isdisJoint = 'false'
            break
if(isdisJoint == 'true'):
    print('two arrays are disjoint')
else:
    print('two arrays are not disjoint')
    